AdobeMediaGalleryApi/Model/Asset/Command/GetByIdInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeMediaGalleryApi\Model\Asset\Command;
```

Comment:
Please move the service interfaces to the `Api` namespace of the module

Reply:
I think that in this case, we can place commands under the Magento\AdobeMediaGallertApi\Model namespace. Those commands interfaces are not the WEB API and according to the [6.4. Service Contracts (Application) layer](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html#64-service-contracts-application-layer): `6.4.1.3. All other APIs, including explicit extension points such as Chain or Composite implementations, MUST be placed under MyModuleApi/Model`. What do you think about this?